sip-core/src/main/java/de/ikor/sip/foundation/core/actuator/routes/MarkdownFileContributor.java

```diff
@@ -0,0 +1,97 @@
+package de.ikor.sip.foundation.core.actuator.routes;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import lombok.SneakyThrows;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.info.Info;
+import org.springframework.boot.actuate.info.InfoContributor;
+import org.springframework.stereotype.Component;
+
+/**
+ * {@link MarkdownFileContributor} extends {@link InfoContributor} to add needed information to the
+ * info actuator
+ */
+@Component
+public class MarkdownFileContributor implements InfoContributor {
+
+  private static final Logger logger = LoggerFactory.getLogger(MarkdownFileContributor.class);
+
+  /**
+   * Entry point. {@link MarkdownObject} List of MarkdownObjects with its informtion is crated and
+   * added to the actuator Info.Builder
+   *
+   * @param builder Info.Builder of the SpringBoot actuator specification
+   */
+  @SneakyThrows
+  @Override
+  public void contribute(Info.Builder builder) {
+
+    ArrayList<MarkdownObject> files = new ArrayList<>();
+
+    MarkdownObject readMe = new MarkdownObject("README.md", "Read me", "");
+    MarkdownObject changeLog = new MarkdownObject("changelog.md", "Change log", "");
+
+    files.add(readMe);
+    files.add(changeLog);
+
+    ArrayList<MarkdownObject> mdFiles = new ArrayList<>();
+
+    for (MarkdownObject obj : files) {
+
+      addMdContentValue(obj);
+      mdFiles.add(obj);
+    }
+
+    builder.withDetail("files", mdFiles);
+  }
+
+  /**
+   * Logic fills the object with mdContent if the file exists on the Classpath. Otherwise it is
+   * null.
+   *
+   * @param mdObj - {@link MarkdownObject} is filled with mdContent if the file exists on the
+   *     Classpath
+   */
+  private void addMdContentValue(MarkdownObject mdObj) {
+
+    InputStream is = getClass().getClassLoader().getResourceAsStream(mdObj.getFileName());
+
+    String data;
+    try {
+      data = readFromInputStream(is);
```

Comment:
I don't see it as a good practice to pass InputStream as parameter. Streams should be handled in one place, meaning opened, consumed and closed.
In this example, readFromInputStream method uses try-with-resources approach, which ensures that input stream will be closed after consumption. But this is not obvious when looking from this point.
Than later down in line 73, we close the is, and it might seam reasonable, bur the stream should already be closed. 

Suggestion:
readFromInputStream method should be renamed, and should take String fileName as input parameter and return file content

Reply:
Apache Commons IOUtils.toString does exactly the same.
We could even use Commons, there are many valuables in it.